src/main/java/com/notably/logic/parser/suggestion/OpenSuggestionCommandParser.java

```diff
@@ -14,9 +17,11 @@
 import com.notably.model.Model;
 
 /**
- * Represents a Parser for OpenSuggestionCommand.
+ * Represents a Parser for DeleteSuggestionCommand.
  */
-public class OpenSuggestionCommandParser implements SuggestionCommandParser<OpenSuggestionCommand> {
+public class OpenSuggestionCommandParser implements SuggestionCommandParser<SuggestionCommand> {
```

Comment:
This should be `implements SuggestionCommandParser<OpenSuggestionCommandParser>`, I believe?